server/index.js

```diff
@@ -91,7 +91,7 @@ const startListening = () => {
   require('./socket')(io)
 }
 
-const syncDb = () => db.sync()
+const syncDb = () => db.sync({force: true})
```

Comment:
This is going to drop everything in your database whenever you start your server over again! Are you sure you want this?

Reply:
Oops. I removed that.